src/tike/operators/ptycho.py

```diff
@@ -33,14 +30,27 @@ class Ptycho(Operator):
         The pixel width and height of the reconstructed grid.
     ntheta : int
         The number of angular partitions of the data.
+    model : string
+        The type of noise model to use for the cost functions.
+    propagation : Operator
+        The wave propagation operator being used.
+    diffraction : Operator
+        The object probe interaction operator being used.
```

Comment:
diffraction may not be the right word for this. or maybe I misunderstood what this operator is.

Reply:
I think I meant "refraction"? The operator that takes a wavefront and an object, then returns the wavefront exiting the object.